packages/QueryEngine/src/Core/Engine/GuardEvaluator.php

```
@@ -7,11 +7,28 @@
 use Nexus\QueryEngine\Exceptions\GuardConditionFailedException;
 
+interface ClockInterface
+{
+    public function now(): \DateTimeImmutable;
+}
+
+final class DefaultClock implements ClockInterface
+{
+    public function now(): \DateTimeImmutable
+    {
+        return new \DateTimeImmutable();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '19', column '20'). (undefined)

(MissingImport)